Retrovizor.Domain/Repositories/Implementations/StudentExamRepository.cs

```diff
@@ -0,0 +1,73 @@
+﻿using Retrovizor.Data.Entities;
+using Retrovizor.Data.Entities.Models;
+using Retrovizor.Domain.Repositories.Interfaces;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace Retrovizor.Domain.Repositories.Implementations
+{
+    public class StudentExamRepository : IStudentExamRepository
+    {
+        StudentExamRepository(RetrovizorContext context)
+        {
+            _context = context;
+        }
+        private readonly RetrovizorContext _context;
+
+        public List<StudentExam> GetAllStudentExams()
+        {
+            return _context.StudentExams.ToList();
+        }
+        public bool AddStudentExam(StudentExam studentExamToAdd)
+        {
+            var doesStudentExamExist = _context.StudentExams.Any(studentExam =>
+                studentExam.StudentId == studentExamToAdd.StudentId && studentExam.ExamId == studentExamToAdd.ExamId);
+
+            if(doesStudentExamExist)
+                return false;
+
+            _context.StudentExams.Add(studentExamToAdd);
+            _context.SaveChanges();
+            return true;
+        }
+        public bool EditStudentExam(StudentExam editedStudentExam)
```

Comment:
ista stvar ka i za add. to bi bilo korisno izdvojit u svoji metodu da se moze korisit na vise mista

Reply:
isto ka za add.